sidechain/consensus/common/src/error.rs

```diff
@@ -53,8 +55,8 @@ pub enum Error {
 	BadParentchainBlock(ParentchainBlockHash, String),
 	#[error("Bad sidechain block (Hash={0}). Reason: {1}")]
 	BadSidechainBlock(SidechainBlockHash, String),
-	#[error("Could not import new block due to {1}. (Last imported by number: {0:?})")]
-	BlockAncestryMismatch(BlockNumber, String),
+	#[error("Could not import new block due to {2}. (Last imported by number: {0:?})")]
+	BlockAncestryMismatch(BlockNumber, SidechainBlockHash, String),
```

Comment:
Hmm, shall we even keep the block number here?

Reply:
I kept it for the error information, it's easier to understand what's going on with block numbers.